distribution/ddf-common/src/main/resources/etc/templates/example.ManagedServiceFactory-2.config

```diff
@@ -1,22 +1,15 @@
-service.factoryPid="example.ManagedServiceFactory"
 property.string="string"
 property.boolean.true=B"true"
 property.boolean.false=B"false"
 property.int=I"10"
 property.long=L"100"
-property.float=F"10.5"
-property.double=D"100.1234"
```

Comment:
@emmberk reminder to add back floats and doubles in this file, or maybe even delete it if it's not being used